tests/test_json_encoder.py

```diff
@@ -0,0 +1,15 @@
+from flask.json import JSONEncoder
+
+
+def test_encoder_html():
+    class ObjectWithHTML:
+        TEST_HTML = "<p>test</p>"
+
+        def __html__(self):
+            return self.TEST_HTML
+
+    encoder = JSONEncoder()
+    assert isinstance(encoder, JSONEncoder)
+    encoder_default = encoder.default(ObjectWithHTML())
+    assert isinstance(encoder_default, str)
+    assert encoder_default == ObjectWithHTML().TEST_HTML
```

Comment:
Just use a string, no need for a class attribute. (Also, since it's a class attribute you wouldn't need to make a new instance to get it.)